tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -470,7 +431,7 @@ def translate_s3_import_to_partitioned_data_sql(report_table_name: str) -> TextC
             SELECT :job_id, orca_archive_location, key_path, etag, last_update, size_in_bytes, storage_class, delete_flag
             FROM s3_import
             WHERE is_latest = TRUE
-        """
+        """  # nosec
```

Comment:
We should review this SQL, not sure why we are passing a variable `{report_table_name}` the table name is a known `reconcile_s3_object`

Reply:
Removed

Reply:
Whoops, didn't see that it was in this file. Fixed.